cfme/infrastructure/host.py

```diff
@@ -130,7 +131,11 @@ def delete(self, cancel=True):
             cancel (bool): Whether to cancel the deletion, defaults to True
         """
         view = navigate_to(self, "Details")
-        view.toolbar.configuration.item_select("Remove item", handle_alert=not cancel)
+        remove_item = VersionPick({
+            '5.8': 'Remove item',
+            '5.9': 'Remove item from Inventory'}).pick(self.appliance.version)
```

Comment:
**QUESTION**
@mfalesni `VersionPick` lives only in widgetastic views and widgets, shouldn't it?

Reply:
@mkoura, as far as I remember VersionPick will be resolved automatically in item_select. So, you don't need this pick at all.

Reply:
my bad, it isn't resolved. I could add a fix if you want